### PR TITLE
feat: Add sweatswap IDO

### PR DESCRIPTION
**Pull request by saltPink**

### CODE REVIEW

File: src/views/Ifos/components/IfoCard/index.tsx [hunk 1]
```diff
@@ -209,34 +209,34 @@ const IfoCard: React.FC<IfoCardProps> = ({ ifo }) => {
           secondsUntilEnd={state.secondsUntilEnd}
           finalized
         />
+        <Flex justifyContent="space-between">
```
Comment (saltKosher) [inline src/views/Ifos/components/IfoCard/index.tsx:212]: Why the changes on this file?

Comment (saltPink) [reply src/views/Ifos/components/IfoCard/index.tsx:212]: So that the IDO info is visible before the IDO
